rules/no-null.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+
+// The number of chars in "Object.create("
+const objectCreateCharCount = 14;
+const objectCreate = 'Object.create(null';
+// Check if the 14 chars preceding `null` are "Object.create("
+const isObjectCreate = (context, node) => context.getSource(node, objectCreateCharCount) === objectCreate;
```

Comment:
This is bad idea 

```js
Object.create( null );
```

```js
Object.create(
null
);
```

```js
Object
  .create(null);
```

Should use selector or check parent.

Reply:
True, it felt a bit hacky. What is the ESLint plugin API for getting a parent node?